lib/tom_queue/delayed_job/job.rb

```diff
@@ -360,16 +360,6 @@ def self.reserve(worker, max_run_time = Delayed::Worker.max_run_time)
       # Returns nil or TomQueue::Work object
       attr_accessor :tomqueue_work
 
-      # Internal: This wraps the job invocation with an acknowledgement of the original
-      # TomQueue work object, if one is around.
-      #
-      def invoke_job
```

Comment:
We used to hook into here - https://github.com/collectiveidea/delayed_job/blob/71f1d5faf934d3057abca942f0d410327bc69087/lib/delayed/worker.rb#L225-L226 be over-riding `invoke_job`